varnishgather

```diff
@@ -353,6 +353,29 @@ _EOF_
 	exit $1
 }
 
+get_vcs_status() {
+# read parameters in unit file
+vcs_override=$( find /etc/systemd/system/vcs.service.d/ | grep ".conf")
+	if [ -z $vcs_override ]; then
+		auth_file=$(grep -o "\-K[[:space:]]*.*" /usr/lib/systemd/system/vcs.service | grep -o "/.*")
+		port=$(grep -o "\-a[[:space:]]*[0-9]*" /usr/lib/systemd/system/vcs.service | grep -o "[0-9]*")
+	else
+		port=$(grep -o "\-a[[:space:]]*[0-9]*" $vcs_override | grep -o "[0-9]*")
+		auth_file=$(grep -o "\-K[[:space:]]*.*" $vcs_override | grep -o "/.*")
+	fi
+	# if port is not modified, set to default 6555
+	if [ -z "$port" ]; then
+		port=6555
+	fi
+
+	if [ -z $auth_file ]; then
+		echo $(curl "http://localhost:"$port"/status")
+	else
+		credentials=$(cat $auth_file)
+		echo $(curl -u $credentials "http://localhost:"$port"/status")
+	fi
```

Comment:
Why `echo` when you could simply let `curl`'s output pass through?

Also consider `curl -s` to avoid adding noise to the output with progress information.